megalinter/descriptors/repository.megalinter-descriptor.yml

```diff
@@ -415,3 +422,58 @@ linters:
       vscode:
         - name: VSCode Trivy
           url: https://marketplace.visualstudio.com/items?itemName=AquaSecurityOfficial.trivy-vulnerability-scanner
+    lint_all_files: true
+
+  # LYCHEE
+  - class: LycheeLinter
+    linter_name: lychee
+    # name: REPOSITORY_LYCHEE
+    can_output_sarif: false
+    linter_url: https://lychee.cli.rs
+    linter_repo: https://github.com/lycheeverse/lychee
+    linter_banner_image_url: https://raw.githubusercontent.com/lycheeverse/lychee/master/assets/banner.svg
+    linter_image_url: https://raw.githubusercontent.com/lycheeverse/lychee/master/assets/logo.svg
+    linter_rules_url: https://lychee.cli.rs/#/usage/cli
+    linter_rules_configuration_url: https://lychee.cli.rs/#/usage/config
+    linter_speed: 2
+    file_extensions:
+      - .md
+      - .MARKDOWN
+      - .html
+      - .htm
+      - .rst
+      - .txt
+      - .json
+      - .yaml
+      - .yml
+      - .epub
+    config_file_name: lychee.toml
+    cli_config_arg_name: --config
+    cli_help_arg_name: --help
+    cli_version_arg_name: --version
+    cli_lint_mode: list_of_files
+    cli_lint_errors_count: regex_number
+    cli_lint_errors_regex: "Errors\\.+([0-9]+)"
+    cli_lint_extra_args:
+      - --format
+      - detailed
+      - --no-progress
+    examples:
+      - lychee --format detailed --no-progress README.md info.txt test.html
+      - lychee --format detailed --no-progress README.md
+      - lychee --format detailed --no-progress test.html info.txt
+      - lychee --format detailed --no-progress --offline path/to/directory
+      - lychee --format detailed --no-progress https://raw.githubusercontent.com/lycheeverse/lychee/master/README.md
+      - 'lychee --format detailed --no-progress "~/projects/big_project/**/README.*"'
+      - 'lychee --format detailed --no-progress --glob-ignore-case --verbose "~/projects/**/[r]eadme.*"'
+    install:
+      apk:
+        - wget
+      dockerfile:
+        - |
+          RUN wget --tries=5 -4 -q -O - https://github.com/lycheeverse/lychee/releases/download/v0.13.0/lychee-v0.13.0-x86_64-unknown-linux-musl.tar.gz | tar -xz lychee \
+            && chmod +x lychee \
+            && mv -f lychee /usr/bin/lychee
+        # - FROM lycheeverse/lychee:0.13.0 as lychee
+        # - COPY --link --from=lychee /usr/local/bin/lychee /usr/bin
```

Comment:
Installing from Docker didn't work that's why?

Reply:
@echoix and this is a great Q... it was my first choice, and always failing.
src of lychee's dockerfile: https://github.com/lycheeverse/lychee/blob/master/Dockerfile

Reply:
This is a ubuntu based image, we probably need an alpine based image to use it as installation instruction

Reply:
Ah, it's because it's because of the linker used. The Dockerfile used is rust-latest, and the last stage is Debian-based.
Thus, it would be the gnu variant instead of the musl-based target, that would be needed in an Alpine-based container. But they release binaries for all of them, so it's a correct way. You can remove the docker comments for now, since unless they change their docker release workflow, it won't help.
```suggestion
```

Reply:
@nvuillam they do not release alpine version officially :( https://hub.docker.com/r/lycheeverse/lychee

Reply:
Ya I saw, it's not a big deal neither. They are close to being able to install with binstall, they have everything there. It's not a requirement for megalinter, every environment has different standards.

Reply:
Oh look, it's already on the way: https://github.com/lycheeverse/lychee/pull/1073
I quickly suggested it in https://github.com/lycheeverse/lychee/issues/59#issuecomment-1555966155

Reply:
@echoix alpine is on the way too https://github.com/lycheeverse/lychee/pull/1074 :D
...it always happened to me 😀, just want to do something what supposed to be quick and easy and finally end up with PRs chain across multiple projects 😅